src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets

```diff
@@ -35,19 +35,21 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <ResolveAssemblyReferencesDependsOn>
+    <ResolveAssemblyReferencesDependsOn Condition="'$(_BuildAgainstPackages)' != 'true'">
       $(ResolveAssemblyReferencesDependsOn);
       ResolveNuGetPackages;
       ValidatePackageVersions;
     </ResolveAssemblyReferencesDependsOn>
+ </PropertyGroup>
 
+ <Import Condition="'$(_BuildAgainstPackages)' == 'true'" Project="buildagainstpackages.targets" />
```

Comment:
It's not.  "BuildTestsAgainstPackages" is the property set via the command line.  When traversing via dir.traversal.targets, though, the property only applies to some projects (test projects), so we need an internal switch to determine if the project we are evaluating is actually applicable to the property or not.